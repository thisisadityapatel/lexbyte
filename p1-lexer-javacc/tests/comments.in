// test simple comments 
#number//followed by a comment
// comment // after comment
?BYEbyebyebool
