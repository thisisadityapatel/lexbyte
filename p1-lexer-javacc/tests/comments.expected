String scanned: [#number]
	->Token kind: <IDNUM>
	->Token value: "#number"
String scanned: [?BYEbyebyebool]
	->Token kind: <IDBOOL>
	->Token value: "?BYEbyebyebool"
